### PR TITLE
Garnett templates typeface

### PR DESCRIPTION
**Pull request by duarte**
This removes fonts from mobile apps templates, and is a first pass at updating text styles throughout.

### CODE REVIEW

File: ArticleTemplates/commentsTemplatePreview.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="initial-scale=1, maximum-scale=1" />
     <link rel="stylesheet" type="text/css" media="all" id="fontStyles" href="__TEMPLATES_DIRECTORY__assets/css/fonts-__PLATFORM__.css" />
-    <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/style-sync.css" />
+    <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/__SYNC_STYLES__.css" />
     <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/style-async.css" />
```
Comment (GHaberis) [inline ArticleTemplates/commentsTemplatePreview.html]: Looks like we need to replace `style-async.css` on line 9 with `__ASYNC_STYLES__`.

Comment (GHaberis) [inline ArticleTemplates/commentsTemplatePreview.html]: Looks like we need to replace style-async.css on line 9 with __ASYNC_STYLES__.

File: ArticleTemplates/footballFixturesTemplate.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1">
     <link rel="stylesheet" type="text/css" media="all" id="fontStyles" href="__TEMPLATES_DIRECTORY__assets/css/fonts-__PLATFORM__.css" />
-    <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/style-sync.css" />
+    <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/__SYNC_STYLES__.css" />
     <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/style-async.css" />
```
Comment (GHaberis) [inline ArticleTemplates/footballFixturesTemplate.html]: Looks like we also need to replace `style-async.css` on line 9 with `__ASYNC_STYLES__` in this template too.

File: ArticleTemplates/simpleBodyTemplate.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="initial-scale=1, maximum-scale=1" />
     <link rel="stylesheet" type="text/css" media="all" id="fontStyles" href="__TEMPLATES_DIRECTORY__assets/css/fonts-__PLATFORM__.css" />
-    <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/style-sync.css" />
+    <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/__SYNC_STYLES__.css" />
     <link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/style-async.css" />
```
Comment (GHaberis) [inline ArticleTemplates/simpleBodyTemplate.html]: Same things as before, replace `style-async.css` on line 9 with `__ASYNC_STYLES__`...

**Comment by GHaberis** [comment]
@duarte this line `<link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/style-async.css" />` should be replaced with `<link rel="stylesheet" type="text/css" media="all" href="__TEMPLATES_DIRECTORY__assets/css/__ASYNC_STYLES__.css" />` wherever it occurs in an HTML template.

**Comment by alfavata** [comment]
Just to double check (as that was my action following a catch-up), the changes @GHaberis requested mean that all the templates will keep working with the platform Garnett switch both disabled (old behaviour) and enabled (new behaviour), right?

**Comment by GHaberis** [comment]
@alfavata yes if the switch is disabled we'll use the old/current css, so the templates will looks as they do now. When you enable the switch we'll use the new CSS and get the updated styles.

**Pull request merged**